src/templates/components/widgets.nunj

```diff
@@ -0,0 +1,25 @@
+{% import "atoms.widgetsmacro.nunj" as widgetsmacro %}
+
+{% set arr = {
```

Comment:
You call the variable `arr`, but it's not an Array, it's an Object.
Maybe use `data` instead?

```diff
- {% set arr = {
+ {% set data = {
```